parlai/zoo/bart/build.py

```diff
@@ -57,14 +65,18 @@ def download(datapath, version='v1.0'):
         build_data.make_dir(dpath)
 
         # Download the data.
-        model_name = 'bart.large'
-        url = f'http://dl.fbaipublicfiles.com/fairseq/models/{model_name}.tar.gz'
-        build_data.download(url, dpath, f'{model_name}.tar.gz')
-        build_data.untar(dpath, f'{model_name}.tar.gz')
-        args = CONVERSION_ARGS.copy()
-        args['input'] = [os.path.join(dpath, model_name, 'model.pt')]
-        args['output'] = os.path.join(dpath, model_name.replace('.', '_'), 'model')
-        ConversionScript.main(**args)
+        models = ['bart.large', 'bart.base']
+        for model_name in models:
+            url = f'http://dl.fbaipublicfiles.com/fairseq/models/{model_name}.tar.gz'
+            build_data.download(url, dpath, f'{model_name}.tar.gz')
+            build_data.untar(dpath, f'{model_name}.tar.gz')
+            args = CONVERSION_ARGS.copy()
+            if model_name == 'bart.base':
+                args['model'] = 'bart/base'
+            args['orig_dict_file'] = os.path.join(dpath, 'bart.large/dict.txt')
```

Comment:
should this be f`{model_name}/dict.txt'? if not, i'm not sure the added option is necessary (since it doesn't seem to differ for either model)